### ISSUE TITLE
Pre-trained Model Availability

### ISSUE THREAD

**Issue by soft-jazz** [facebookresearch/dlrm#40]
Hello,

Are there any pre-trained models that we can use for benchmark testing / evaluation? I'm trying to reproduce the results from section 5.1/5.2 of the [original DLRM paper](https://arxiv.org/pdf/1906.00091.pdf) and it would be useful to have a pre-trained model available to do speed / accuracy tradeoff comparisons for a class project. 

Thanks!

**Comment by mnaumovfb** [comment]
Unfortunately, we do not provide a pre-trained model. I will take this as a request for the future.

You should be able to train a model from scratch in a few hours when interfacing with Criteo Kaggle Display Advertising Challenge Dataset. Then, you can use --enable-profiling flag to collect the required statistics and reproduce the paper results.

**Comment by mnaumovfb** [comment]
I'm assuming this is resolved. Closing.
